src/evo/simplifiedmns.cpp

```diff
@@ -92,13 +93,17 @@ CSimplifiedMNList::CSimplifiedMNList(const std::vector<CSimplifiedMNListEntry>&
     });
 }
 
-CSimplifiedMNList::CSimplifiedMNList(const CDeterministicMNList& dmnList)
+CSimplifiedMNList::CSimplifiedMNList(const CDeterministicMNList& dmnList, const CBlockIndex* pindexPrev)
 {
     mnList.resize(dmnList.GetAllMNsCount());
 
+    bool v19active = llmq::utils::IsV19Active(pindexPrev);
```

Comment:
I don't think this is a good spot to be doing the check.. Why don't we pass in a boolean here representing whether v19 is active? that way we don't have to include llmq/utils here as well.

Reply:
Sure